src/docs/assets/stackblitz/src/styles.css

```diff
@@ -0,0 +1,15 @@
+body {
+  margin: 0;
+  padding: 30px;
+}
+
+html,
+body {
+  height: 100%;
+}
+
+.version-info {
```

Comment:
please use a lyne class for text size. e.g sbb-text-xs. and for other values in this file, use design tokens